vendor/assets/stylesheets/multi-select.scss

```diff
@@ -100,27 +97,22 @@
   cursor: text;
 }
 .ms-header{
-    font-weight: bold;
-    color:#ffffff;
-    background-color: #aaa;
-    padding: 4px 10px;
-    -webkit-border-radius: 4px 4px 0 0;
-    -moz-border-radius: 4px 4px 0 0;
-    border-radius: 4px 4px 0 0;
+  padding: 4px 10px;
+  border: 1px solid #d1d1d1;
+  background-clip: padding-box;
+  background-color: #f9f9f9;
+  background-image: -webkit-linear-gradient(top, #fafafa 0%, #ededed 100%);
+  background-image: linear-gradient(to bottom, #fafafa 0%, #ededed 100%);
+  background-repeat: repeat-x;
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#fffafafa', endColorstr='#ffededed', GradientType=0);
```

Comment:
just put `@import "mixin";` at the top. 
While you're at it, we also have mixin for box-shadow, and transition no longer needs all the browser prefixes (http://caniuse.com/#feat=css-transitions) for any modern browser